string target_ship
float64 target_yaw
float64 target_pitch
float64 max_speed_rocket
